content/en/docs/reference/command-line-tools-reference/feature-gates/all-dns-only-node-csr.md

```diff
@@ -0,0 +1,14 @@
+---
+title: AllDNSOnlyNodeCSR
```

Comment:
_Quick question:_  Is the feature gate name `AllDNSOnlyNodeCSR` correct? I couldn’t find any references to it in the upsteam [k/k ](https://github.com/kubernetes/kubernetes)codebase, so just wanted to double-check.

Reply:
I think I may have found it; it’s named `AllowDNSOnlyNodeCSR`. 

_**Code [reference link (here)](https://github.com/kubernetes/kubernetes/blob/6d1bcd738fe49754c9f4dc53eac634eab41e0967/pkg/features/kube_features.go#L53)**_
```go
	// Deprecated: v1.31
	//
	// Allow kubelet to request a certificate without any Node IP available, only
	// with DNS names.
	AllowDNSOnlyNodeCSR featuregate.Feature = "AllowDNSOnlyNodeCSR"
```


This would also mean a change in the name of this file.
```suggestion
title: AllowDNSOnlyNodeCSR
```

Reply:
D'oh. Ta!